dataset/src/test/scala/frameless/TypedDatasetSuite.scala

```diff
@@ -6,11 +6,7 @@ import org.scalactic.anyvals.PosZInt
 import org.scalatest.FunSuite
 import org.scalatest.prop.Checkers
 
```

Comment:
@jeremyrsmith, I don't mind this at all. Just wondering what triggered you to do this change?